diff --git a/node_modules/@strapi/plugin-content-type-builder/dist/server/index.js b/node_modules/@strapi/plugin-content-type-builder/dist/server/index.js
index ae15d81..a1d6564 100644
--- a/node_modules/@strapi/plugin-content-type-builder/dist/server/index.js
+++ b/node_modules/@strapi/plugin-content-type-builder/dist/server/index.js
@@ -1530,12 +1530,14 @@ const getTypeShape = (attribute, { modelType, attributes } = {}) => {
             const targetCompo = strapi.components[compoUID];
             if (!targetCompo)
               return true;
-            if (modelType === modelTypes.COMPONENT && hasComponent(targetCompo)) {
-              return this.createError({
-                path: this.path,
-                message: `${targetCompo.modelName} already is a nested component. You cannot have more than one level of nesting inside your components.`
-              });
-            }
+            // TODO: We need to disable this check for now to allow nested components which are already created in project
+
+            // if (modelType === modelTypes.COMPONENT && hasComponent(targetCompo)) {
+            //   return this.createError({
+            //     path: this.path,
+            //     message: `${targetCompo.modelName} already is a nested component. You cannot have more than one level of nesting inside your components.`
+            //   });
+            // }
             return true;
           }
         }).required(),
